docs/05_operations/12_logging/01_logging.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Logging'
+sidebar_label: 'Logging'
+id: tracing
+keywords: [operations, logging]
+tags:
+    - operations
+    - logging
+---
+
+This page relates to Server logging.
+
+## Logging level
+
+The logging level can be modified in the processes xml file. See [here](https://learn.genesis.global/docs/server/configuring-runtime/processes/#logginglevel) for more details.
```

Comment:
why use an absolute path rather than a relative one? couldn't this bring about issues a few versions down the line? the link would be referencing the at the time current page which may no longer be relevant to this version

Reply:
Oh damn, good point. Definitely didn't mean to use full http url!